commands/scan/dockerscan.go

```diff
@@ -63,17 +68,19 @@ func (dsc *DockerScanCommand) Run() (err error) {
 	}()
 
 	// Run the 'docker save' command, to create tar file from the docker image, and pass it to the indexer-app
-	if dsc.progress != nil {
-		dsc.progress.SetHeadlineMsg("Creating image archive 📦")
-	}
-	log.Info("Creating image archive...")
-	imageTarPath := filepath.Join(tempDirPath, "image.tar")
-	dockerSaveCmd := exec.Command("docker", "save", dsc.imageTag, "-o", imageTarPath)
-	var stderr bytes.Buffer
-	dockerSaveCmd.Stderr = &stderr
-	err = dockerSaveCmd.Run()
-	if err != nil {
-		return fmt.Errorf("failed running command: '%s' with error: %s - %s", strings.Join(dockerSaveCmd.Args, " "), err.Error(), stderr.String())
+	imageTarPath := dsc.imageTag
+	if !dsc.isTar {
+		if dsc.progress != nil {
```

Comment:
```suggestion
        skipSave := dsc.isTar || (fileutils.IsFileExists(dsc.imageTag) && strings.HasSuffix(".tar"))
		if !skipSave {
```
I would add a test to check if this given argument is an existing file path (with .tar suffix) or if this flag is provided, to force skiping.